ui/desktop/app/routes/scopes/scope/projects/sessions/index.js

```diff
@@ -133,11 +144,17 @@ export default class ScopesScopeProjectsSessionsIndexRoute extends Route {
   }
 
   resetController(controller, isExiting, transition) {
-    const { to } = transition;
+    const { scopes, targets, status } = transition.to.queryParams;
     // Reset the scopes query param when changing org scope
-    if (!isExiting && to.queryParams.scopes === '[]') {
+    if (isExiting && (scopes || targets || status)) {
```

Comment:
These parameters are going to be strings correct? i.e if it's an empty array it will be `"[]"`? Is this the intent?

Reply:
Absolutely correct. Thanks for catching that. I will update now.